dandi/model_types.py

```diff
@@ -573,3 +573,21 @@
         },
     ]
 }
+
+AgeReferenceTypeDict = {
+    "@graph": [
+        {
+            "@id": "dandi:AgeReferenceType",
+            "@type": "rdfs:Class",
+            "rdfs:comment": "An enumeration of age reference",
+            "rdfs:label": "Age reference type",
+            "rdfs:subClassOf": {"@id": "schema:Enumeration"},
+        },
+        {
+            "@id": "dandi:BirthReference",
+            "@type": "dandi:AgeReferenceType",
+            "rdfs:comment": "Age since Birth",
+            "rdfs:label": "BirthReference",
+        },
```

Comment:
also add a gestational age reference. you will need that for some datasets too: https://en.wikipedia.org/wiki/Gestational_age

Reply:
should I be adding pointers to wikipedia or other sources, or the short description is enough: https://github.com/dandi/dandi-cli/pull/450/commits/aaa1973d5f71870db058f3684caaa909601b8602

Reply:
always useful to add references when possible - you can add it to the comment field.